client/client-multi/client-objective-c/Kaa/channel/impl/channels/DefaultOperationTcpChannel.m

```diff
@@ -278,14 +269,18 @@ - (void)stream:(NSStream *)aStream handleEvent:(NSStreamEvent)eventCode {
     switch (eventCode) {
         case NSStreamEventOpenCompleted:
         {
+            self.isPingTaskCancelled = NO;
```

Comment:
You should set it to NO each time you schedule a new ping task. (Not execute but schedule)